src/components/Filter/Filter.jsx

```diff
@@ -0,0 +1,36 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { Link } from 'react-router-dom';
+
+export const Filter = ({ setFilter, filters }) => (
+  <ul className="filters">
+    <Link
+      to="/"
+      className="basic"
+      onClick={() => setFilter(filters.all)}
```

Comment:
```suggestion
      onClick={() => setFilter(filters.ALL)}
```
let's name constants in upper case